docs/usage.md

```diff
@@ -6,48 +6,45 @@
 
 ## Introduction
 
-The pipeline can either be run with individual parameters specifying input files and evolutionary model, or from a samplesheet.
-The latter allows you to place more than one set of query sequences in reference trees.
+This pipeline performs "phylogenetic placement" of query sequences on to a reference phylogeny.
+With this method, the likelihoods of placing query sequences in the reference phylogeny are calculated on a one-by-one basis.
+This means that a full phylogeny is not estimated, which is perfect e.g. when sequences come from PCR amplicons or are fragments from metagenomes, but also when the number of query sequences is very large.
+One of the outputs from the pipeline is nevertheless a full phylogeny containing both query and reference sequences, but this is produced by grafting the query sequences on to the reference phylogeny in their most likely positions.
 
-## Samplesheet input
+Placement is performed with the [EPA-NG](https://github.com/Pbdas/epa-ng/blob/master/README.md) program after query sequences have been aligned to the reference alignment.
+
+At minimum, four parameters are required: A fasta file with query sequences, a file with a reference alignment (several popular formats supported), a reference phylogeny and a phylogenetic model.
```

Comment:
I think it would help tremendously to name to the params and turn to list, i.e.
```suggestion
At minimum, four parameters are required: 
- `--queryfile`: a fasta file with query sequences, 
- `--refalignment`: a file with a reference alignment (several popular formats supported), 
- `--refphylogeny`: a reference phylogeny and 
- `--model`: a phylogenetic model.
```
caution: this suggestion might upset prettier linting ;)

Reply:
Did something like this.